### DISCUSSION TITLE
Hide live chat replay

### DISCUSSION THREAD

**Discussion by DarkblooM-IO** [Pedro-Gregorio/TubeMod#78] [Ideas]
Can you add an option to hide the live chat replay section underneath the video description?
![image](https://github.com/user-attachments/assets/b17866a7-87b5-4eb8-9885-941fc461d85a)

**Comment by Pedro-Gregorio** [comment]
Hello there! Yeah, this thing is new and annoys me 😄 
Should come out on the next version, have it planned!

**Reply by DarkblooM-IO** [reply]
Neat!
